src/components/PurchaseOrder/PODetails/PODetailsForm.js

```diff
@@ -271,10 +296,17 @@ class PODetailsForm extends Component {
               name="ship_to"
             />
           </Col>
-          <Col xs={6} md={3}>
+          <Col
+            xs={6}
+            lg={3}
+          >
             <FieldOrderType />
           </Col>
-          <Col xs={12}>
+          <Col
+            xs={8}
```

Comment:
don't you think that it's kind of hard to keep in mind all these numbers of `xs`? It's not obvious where field will be on a new line. Maybe it makes sense to break up on lines using `<Row>` tags?